docker_images/sklearn/tests/test_api_tabular_classification.py

```diff
@@ -67,26 +76,90 @@ def test_success_code(self):
         assert type(content) == list
         assert len(content) == expected_output_len
 
-    def test_malformed_input(self):
+    def test_wrong_sklearn_version_warning(self):
+        # if the wrong sklearn version is used the model will be loaded and
+        # gives an output, but warnings are raised. This test makes sure the
+        # right warnings are raised and that the output is included in the
+        # error message.
+        self._check_requirement(self.case_data["old_sklearn"] and self._can_load())
+
+        data = self.data
         with TestClient(self.app) as client:
-            response = client.post("/", data=b"Where do I live ?")
+            response = client.post("/", json={"inputs": data})
 
-        self.assertEqual(
-            response.status_code,
-            400,
-        )
+        assert response.status_code == 400
         content = json.loads(response.content)
-        self.assertEqual(set(content.keys()), {"error"})
+        assert "error" in content
+        assert "warnings" in content
+        wrong_versions = [
+            w for w in content["warnings"] if "Trying to unpickle estimator" in w
+        ]
+        assert len(wrong_versions)
```

Comment:
Are those lines equivalent to:

`assert any("Trying to unpickle estimator" in w for w in content["warnings"])`